### ISSUE TITLE
Magnifier / Screen curtain conflict

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20289]
### Steps to reproduce
1. Start magnifier (NVDA+shift+W)
2. Start permanent screen curtain (double press on `NVDA+control+Escape`)
3. Save your config (`NVDA+control+C`)
4. Restart NVDA

For last beta:
5. Start Magnifier (`NVDA+shift+W`)
6. Press `NVDA+control+escape`

For 2026.2beta1:
5. Disable screen curtain (`NVDA+control+escape`)
6. Start Magnifier (`NVDA+shift+W`)

### Actual behavior
NVDA last beta branch:
* At step 4: just after restart, NVDA briefly shows black screen (screen curtain (and the screen curtain on sound is heard), then shows a freezed zoomed (partial) image of the screen
* At step 5: NVDA reports "Cannot start magnifier, screen curtain is active"
* At step 6: NVDA reports "Screen curtain disabled" (with sound) and the freezed image disappears.

NVDA 2026.2beta1:
* At step 6, magnifier should start; instead, you have an error message and you need to press one more time `NVDA+shift+W` to make it actually start.
### Expected behavior
NVDA should restart with screen curtain enabled.
Then when screen curtain is disabled, NVDA should be able to launch Magnifier at the first time with no error.

### Note
This issue is due to a conflict in magnification API that has not been fixed with #20069. Worse, it has become a privacy issue since a blind user can have their screen visible while they think that the screen curtain is active.

### NVDA logs, crash dumps and other attachments
N/A
#### NVDA installed/portable/running from source
* Installed (2026.2beta1)
* or from source (last beta branch)
#### NVDA version
* Installed 2026.2beta1
* From source: last beta branch (commit b853b449c336f720696339e5154152f9b385b420)
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6783
#### Name and version of other software in use when reproducing the issue
N/A
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors
No, only 2026.2beta1 and last beta branch
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested
